proofs/logic.ml

```diff
@@ -67,6 +67,7 @@ let catchable_exception = function
   | Nametab.GlobalizationError _
   (* reduction errors *)
   | Tacred.ReductionTacticError _ -> true
+  | Logic_monad.TacticFailure _ -> true
```

Comment:
I don't get this change, is it related?

Reply:
Sorry, a collision with another PR I was working on at roughly the same time. Fixed